cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -180,6 +180,24 @@ spec:
       containers:
       - name: "{{ .label }}"
         image: "{{ $root.Values.Image.repository }}:{{ $root.Values.Image.tag }}"
+        {{ if $root.Values.initDb}}
+        {{ if eq .name "yb-tservers" }}
+        lifecycle:
```

Comment:
`{{- $masterAddresses := template "yugabyte.master_addresses" $root  -}}` try the variable route so we don't duplicate the function